File: Java OOP/Homework07/models/Company.py
class Company:
    def __init__(self):
        self.departments = []

    def add(self, department):
        self.departments.append(department)

    def get_departments(self):
        return self.departments

    def get_department(self, index):
        return self.departments[index]

    def remove(self, index):
        self.departments.pop(index)

File: Java OOP/Homework06/models/Company.java
// создай класс как в этом коде, но на Python

package models;

import java.util.ArrayList;

public class Company {
    private ArrayList<Department> departments = new ArrayList<>();

    public void add(Department department) {
        departments.add(department);
    }
  
    public ArrayList<Department> getDepartments() {
        return this.departments;
    }

    public Department getDepartment(int index) {
        return this.departments.get(index);
    }

    public void remove(int index) {
        departments.remove(index);
    }
}
